pkg/traceql/enum_attributes.go

```diff
@@ -64,11 +64,13 @@ const (
 
 	// These intrinsics do not map to specific data points, but are used to
 	// indicate that Spans must be able to answer the structural methods
-	// DescdendantOf, SiblingOf, and ChildOf.  The details of those methods
+	// DescdendantOf, AncestorOf, SiblingOf, ChildOf and ParentOf.  The details of those methods
 	// and how these intrinsics are handled is left to the implementation.
 	IntrinsicStructuralDescendant
+	IntrinsicStructuralAncestor
```

Comment:
I don't think we need these new intrinsics.  For example, for ancestor check we are calling the existing method `span.IsDescendantOf(other)`, but with reversed arguments, we should be able to reuse `IntrinsicStructuralDescendant`.   This way we don't have to make any changes at the storage layer (vParquet2/3 encodings).